portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeAssetEntry.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.upgrade.v7_4_x;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Alberto Chaparro
+ */
+public class UpgradeAssetEntry extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		String inClause = StringPool.BLANK;
+
+		for (String className : _CLASS_NAMES) {
+			long classNameId = _getClassNameId(
+				_CLASS_NAME_MBDISCUSSION + StringPool.UNDERLINE + className);
+
+			if (classNameId != 0) {
+				inClause += classNameId + ",";
```

Comment:
Can we use a StringBundler here instead of regular string concatenation?

Reply:
Hey @adolfopa, I added the requested changes

Reply:
@achaparro What I meant was to something like this:
```
		StringBundler sb = new StringBundler();

		for (String className : _CLASS_NAMES) {
			long classNameId = _getClassNameId(
				_CLASS_NAME_MBDISCUSSION + StringPool.UNDERLINE + className);

			if (classNameId != 0) {
				sb.append(classNameId);
				sb.append(StringPool.COMMA);
			}
		}

		String inClause = sb.toString();

		if (inClause.equals(StringPool.BLANK)) {
			return;
		}
```

Sorry, I'd do this myself, but I would prefer not to change my local branche right now.